src/software/estop/threaded_estop_reader.h

```diff
@@ -56,8 +57,8 @@ class ThreadedEstopReader
      * any other message received is considered a EstopState::STATUS_ERROR
      */
     static constexpr int ESTOP_MESSAGE_SIZE_BYTES = 1;
-    static constexpr unsigned char ESTOP_PLAY     = 1;
-    static constexpr unsigned char ESTOP_STOP     = 0;
+    static constexpr unsigned char ESTOP_PLAY     = ESTOP_PLAY_MSG;
+    static constexpr unsigned char ESTOP_STOP     = ESTOP_STOP_MSG;
```

Comment:
why can't we directly use ESTOP_PLAY_MSG and ESTOP_STOP_MSG instead?

Reply:
we can, changed